arteria/models/runfolder_utils.py

```diff
@@ -22,26 +22,51 @@ def list_runfolders(monitored_directories, filter_key=lambda r: True):
     runfolders when no state filter is given.
     """
     runfolders = []
-    for monitored_directory in monitored_directories:
-        monitored_dir_path = Path(monitored_directory)
-        for subdir in monitored_dir_path.iterdir():
-            try:
-                if filter_key(runfolder := Runfolder(monitored_dir_path / subdir)):
-                    runfolders.append(runfolder)
-            except AssertionError as e:
-                if e == f"File [Rr]unParameters.xml not found in runfolder {subdir}":
-                    continue
+    monitored_runfolders_paths = get_monitored_path_files(monitored_directories)
```

Comment:
The name `get_monitored_path_files` does not quite reflect what this function does. Maybe it could be `get_monitored_runfolders` instead?